api/v1alpha1/enterprisecontractpolicy_types.go

```diff
@@ -90,6 +90,15 @@ type EnterpriseContractPolicyExceptions struct {
 	NonBlocking []string `json:"nonBlocking,omitempty"`
 }
 
+// EnterpriseContractPolicyConfiguration configuration of modifications to policy evaluation
+type EnterpriseContractPolicyConfiguration struct {
+	// +optional
+	// +listType:=set
+	ExcludeRules []string `json:"excludeRules,omitempty"`
```

Comment:
We could add comments for these fields so they show up in the CRD and docs

Reply:
Added.